<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OnsiteDetailTable</class>
 <widget class="QWidget" name="OnsiteDetailTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLB">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="onsiteTW">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
QTableView::item {
	font-size: 9pt;
} </string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>23</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>30</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>30</number>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>version</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>msg_type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>t_time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>station</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>channel</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>network</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>location</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>duration</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>disp_count</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>displacement</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>disp_time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>pvel</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>pacc</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>tauc</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>pgv</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>pgv_uncertainty_low</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>pgv_uncertaInty_high</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>alert</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>snr</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>log_disp_ratio</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>log_PdPv</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>dB_Pd</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>intensity</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>intensity_uncertainty_low</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>intensity_uncertainty_high</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>magnitude</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>magnitude_uncertainty_low</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>magnitude_uncertainty_high</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>distance</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>distance_uncertainty_low</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>distance_uncertainty_high</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="chartFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>980</width>
       <height>300</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>980</width>
       <height>300</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="chartLO">
        <property name="spacing">
         <number>0</number>
        </property>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
